optscale-deploy/runkube.py

```diff
@@ -321,8 +324,15 @@
         except IndexError:
             LOG.info('etcd pod not found')
 
+    def check_version(self):
+        if self.version.lower() == LATEST_TAG:
+            self.version = subprocess.check_output(
+                GET_LATEST_TAG_CMD, shell=True).decode("utf-8").rstrip()
```

Comment:
I think it's safe to dismiss this alert as `GET_LATEST_TAG_CMD` always evaluates to a static string, so no risk for injection with a user string.

I suppose they added this due to the pipe in the command -- this is is not strictly necessary as the output of `curl` can be parsed in python instead of `jq` but that's how hystax devs implemented it and I'd rather keep it that way to keep future syncs as easy as possible

Reply:
Actually now looking through the changes we've made in our fork (https://github.com/hystax/optscale/compare/integration...softwareone-platform:optscale:main) a lot of them are adding `usedforsecurity=False` in `hashlib.md5` calls - @ffaraone was this done to suppress similar `bandit` issues?

While these are great changes maybe we should rather create PRs to hystax/optscale directly about them, so that:

1. We get fewer merge conflicts whenever we're doing a sync, and
2. We solve the issue at the source, I'm sure Hystax will appreciate this

It's weird our pipeline caught this but not theirs, `bandit` is a very popular and useful tool, so to go even further we could open an issue on their repo to hear their thoughts about integrating it into their pipeline. That way changes like this won't be pushed in the first place which works best for everyone.

@ffaraone @antoniodimariano what do you think?